### ISSUE TITLE
[v3.6.0] Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/../..'

### ISSUE THREAD

**Issue by sectsect** [cascornelissen/svg-spritemap-webpack-plugin#128]
I get console errors on `watch` mode.
☝️ FYI: However, the svg file is generated well.

```
$ npm run dev
...
[webpack-cli] watching files for updates...
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/xxx/xxx/svg-spritemap-webpack-plugin-test/src/assets/svg/facebook.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/xxx/xxx/svg-spritemap-webpack-plugin-test/src/assets/svg/instagram.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/xxx/xxx/svg-spritemap-webpack-plugin-test/src/assets/svg/twitter.svg'
Watchpack Error (initial scan): Error: ENOTDIR: not a directory, scandir '/xxx/xxx/svg-spritemap-webpack-plugin-test/src/assets/svg/youtube.svg'
```

The error can be resolved by changing the glob pattern as shown below.  
Is this the expected behavior?
```diff
- new SVGSpritemapPlugin(path.resolve(sourcePath, 'assets/svg/**/*.svg'), {...}),
+ new SVGSpritemapPlugin(path.resolve(sourcePath, 'assets/svg/*'), {...}),
```

## My Test Project
I created a minimal project.
(webpack: `5.3.2` / svg-spritemap-webpack-plugin: `3.6.0`)
https://github.com/sectsect/svg-spritemap-webpack-plugin-test

1. `npm run dev`
2. Check your console.

**Comment by cascornelissen** [comment]
Interesting, looks like some file dependencies are getting marked as directory dependencies 🤔 Are you sure this does not happen with Webpack 4?

**Comment by cascornelissen** [comment]
Found the issue and a fix is on the way ✌🏼

**Comment by cascornelissen** [comment]
This should be fixed in `3.6.1` which is now available under the `next` dist-tag on NPM.

```
$ npm install svg-spritemap-webpack-plugin@next
```

Please test this release out using both Webpack 4 and 5 if possible and report any problems.
